10.1/Dockerfile.fedora

```diff
@@ -19,14 +19,12 @@ LABEL summary="MariaDB is a multi-user, multi-threaded SQL database server" \
       io.k8s.description="MariaDB is a multi-user, multi-threaded SQL database server" \
       io.k8s.display-name="MariaDB 10.1" \
       io.openshift.expose-services="3306:mysql" \
-      io.openshift.tags="database,mysql,mariadb,mariadb101,galera"
-
-ENV NAME=mariadb VERSION=10.1 RELEASE=1 ARCH=x86_64
```

Comment:
Good point, we can check it and change later if it will work, so let's not do it for now.. I think it might still work during rebuilding, because `release` label value in upstream is ignored IMO.